lib/publiccloud/azure.pm

```diff
@@ -17,8 +17,6 @@ use utils qw(script_output_retry);
 use publiccloud::azure_client;
 
 has resource_group => 'openqa-upload';
-# Note: Remove the default 'openqa' once the deprecated account will be removed.
-has storage_account => get_var('PUBLIC_CLOUD_STORAGE_ACCOUNT', 'openqa');
```

Comment:
Why do we remove this setting here now and not just the default value? This was present before and I think many of the other code changes are thereby unnecessary.

Reply:
I would also need more info for this change. Maybe the `$storage_account` should be a optional parameter?

Reply:
A other thing. Does this actually work and should not be 
```perl
has storage_account => sub { get_var('PUBLIC_CLOUD_STORAGE_ACCOUNT', 'openqa')};
```

Reply:
now we migrated to new account and because storage account names suppose to be unique globally in whole Azure we could not create storage account with `openqa` name anymore this is why we picked up new name